pysbd/lang/slovak.py

```diff
@@ -72,7 +72,8 @@ def __init__(self, text, lang, char_span=False):
         def process(self):
             if not self.text:
                 return self.text
-            self.text = self.text.replace('\n', '\r')
```

Comment:
Toto by nemalo ísť preč.

Reply:
Navrhujem zachovať (lebo to tam imho má byť):

```suggestion
            self.text = self.text.replace('\n', '\r')
```

Reply:
Ok, ponechaj. Škoda normalizovaný by bol lepší.